ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -146,7 +107,8 @@ private static HttpServerOptions applyTlsKeyStore(
     try {
       final String keyStorePathname =
           tlsConfig.getKeyStoreFile().toPath().toAbsolutePath().toString();
-      final String password = readSecretFromFile(tlsConfig.getKeyStorePasswordFile().toPath());
+      final String password =
+          FileUtil.readFirstLineFromFile(tlsConfig.getKeyStorePasswordFile().toPath());
```

Comment:
Why restrict the password to only the first line?

Reply:
Because the use case is as such. The password is meant to be the first (and possibly) only line we are interested in. The previous logic was reading the whole file as a byte array, converting it to String and treating that as a password. (is multi-line password possible though?)

Reply:
miltiline password would be super weird, i've never seen a use case for that.
I'd point out reading more than the first line would open you to people in unix putting password\n
which is the normal file format (passwordEOF will make things like vim unhappy)

Reply:
I thought the password would be whatever is in the file regardless of new line characters so seems odd to me have this restriction. But if you think this makes more sense i'm happy to with that.